metrics-scraper/monitoring.go

```diff
@@ -163,12 +215,14 @@ func sidecarsVersion(ctx context.Context, client *rpc.Client, moniker, networkID
 	result := map[string]net.SidecarInfos{}
```

Comment:
I think we can use the same as proposed in https://github.com/mezo-org/mezod/pull/450#discussion_r2057588722 for `sidecarsVersion`.

Reply:
addressed in: d1755782d3115b584b67e6fb763dc3d62386ef81